assets/js/components/legacy-setup/SetupUsingGCP.js

```diff
@@ -189,32 +199,8 @@ class SetupUsingGCP extends Component {
 		return '';
 	}
 
-	async onButtonClick() {
```

Comment:
This is actually not invoked, the other button was called from `authentication` step in setup wizard, so I moved the logic fir user setup there, and site setup to other step, once the site is connected